amundsen_application/static/js/index.tsx

```diff
@@ -36,7 +36,7 @@ ReactDOM.render(
       <BrowserRouter>
         <div id="main">
           <Preloader/>
-          <NavBar />
+          <Route path="/" component={NavBar} />
```

Comment:
Not entirely sure what's going on here. Wouldn't setting `path="/"` mean the `NavBar` is only rendered on the `/` path? At least that should be the case if it were inside the `Switch`. Does `path` get used at all then?

Reply:
`path` means that a component will get rendered if the path is partial match. (we could use the `exact` option if we wanted exact match for anything by the way). Because `path` is partial match, this pretty much means render the `NavBar` on every page. Which is what we want and what we were doing previously.

However now that you mention it, I will see if it still renders if I only use `<Route component={NavBar} />` without `path`. If that works, I'll change to that.